db/migrate/20210430124303_add_edited_and_deleted_to_histories.rb

```diff
@@ -0,0 +1,19 @@
+class AddEditedAndDeletedToHistories < ActiveRecord::Migration[6.1]
```

Comment:
Looks like this is placed _before_ two other migrations (`RemoveContentsAndLookupFromAndAddMarkedForDeletionToDownloads` and `CreateActiveStorageTables`). I'm guessing that was just the result of a rebase, but this migration should be the most recent.